youtube_dl/extractor/pluralsight.py

```diff
@@ -215,7 +219,7 @@ def _real_extract(self, url):
         for module in course_data:
             for clip in module.get('clips', []):
                 player_parameters = clip.get('playerParameters')
-                if not player_parameters:
+                if not player_parameters or clip.get('userMayViewClip') is False:
```

Comment:
Relying on `userMayViewClip` is probably a [bad idea](https://github.com/rg3/youtube-dl/commit/2b6bda1ed86e1b64242b33c032286dc315d541ae#diff-0b85b33765d2939b773726fda5a55b06).